scipy/diff/statsmodels/_derivative.py

```diff
@@ -0,0 +1,175 @@
+from __future__ import division
+import numpy as np
+from _epsilon_generator import _epsilon
+
+
+def derivative(f, x, **options):
+    '''
+    Derivative of a function
+
+    Parameters
+    ----------
+    x : array
+        parameters at which the derivative is evaluated
+
+    f : function
+        `f(x)` returning one value.
+        `f(x)` should be univariate.
+
+    **options : dict
+        options for specifying the method and epsilon.
+        method : {'central','foward'}.
```

Comment:
ReST stands for ReStructedText, which is what all our docs are written in and what Sphinx understands. See http://www.sphinx-doc.org/en/stable/rest.html